pymc/tests/distributions/test_truncated.py

```diff
@@ -6,25 +6,32 @@
 
 from aeppl.logprob import ParameterValueError, _icdf
 from aeppl.transforms import IntervalTransform
-from aesara.tensor.random.basic import GeometricRV, NormalRV
+from aesara.tensor.random.basic import GeometricRV
 
 from pymc import Censored, Model, draw, find_MAP, logp
-from pymc.distributions.continuous import Exponential, TruncatedNormalRV
+from pymc.distributions.continuous import Exponential, PyMCNormalRV, TruncatedNormalRV
 from pymc.distributions.shape_utils import change_dist_size
 from pymc.distributions.transforms import _default_transform
 from pymc.distributions.truncated import Truncated, TruncatedRV, _truncated
 from pymc.exceptions import TruncationError
 from pymc.tests.distributions.util import assert_moment_is_expected
 
 
-class IcdfNormalRV(NormalRV):
+class IcdfNormalRV(PyMCNormalRV):
     """Normal RV that has icdf but not truncated dispatching"""
 
 
-class RejectionNormalRV(NormalRV):
+class RejectionNormalRV(PyMCNormalRV):
     """Normal RV that has neither icdf nor truncated dispatching."""
 
 
+"""
```

Comment:
I would also do it for the GeometricRV since we also use a PyMCGeometricRV internally. The tests work regardless but I think it's less confusing for future users.